src/components/Shared/ScatterPlot/ScatterPlotComponent.tsx

```diff
@@ -382,14 +356,58 @@ export class ScatterPlotComponent extends React.Component<ScatterPlotComponentPr
                 const cursorXPosGraphSpace = this.getValueForPixelX(mousePosX);
                 const cursorYPosGraphSpace = this.getValueForPixelY(mousePosY);
                 this.props.graphCursorMoved(cursorXPosGraphSpace, cursorYPosGraphSpace);
+            } else if (this.isPanning && this.props.graphZoomedXY) {
+                const currentPanX = mousePosX;
+                const currentPanY = mousePosY;
+                const prevPanGraphSpaceX = this.getValueForPixelX(this.panPrevious.x);
+                const currentPanGraphSpaceX = this.getValueForPixelX(currentPanX);
+                const deltaX = (currentPanGraphSpaceX - prevPanGraphSpaceX);
+                const prevPanGraphSpaceY = this.getValueForPixelY(this.panPrevious.y);
+                const currentPanGraphSpaceY = this.getValueForPixelY(currentPanY);
+                const deltaY = (currentPanGraphSpaceY - prevPanGraphSpaceY);
+                this.updatePan(currentPanX, currentPanY);
+                const pinMinX = this.props.xMin - deltaX;
+                const pinMaxX = this.props.xMax - deltaX;
+                const pinMinY = this.props.yMin - deltaY;
+                const pinMaxY = this.props.yMax - deltaY;
+                this.props.graphZoomedXY(pinMinX, pinMaxX, pinMinY, pinMaxY);
```

Comment:
Just check this code, you might have the Y-axis direction mixed up. When you zoom in using the scroll wheel, it zooms centered on the correct X-coordinate, but the wrong Y-coordinate

Reply:
Thanks, fixed now.